app/views/users/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Users Profile</title>
+  </head>
+  <body>
+    <h1>This is the User profile page</h1>
+  </body>
+</html>
```

Comment:
- [ ] You have no user index view page. Kindly create one in this directory to be `app/views/users/index.html.erb`